### ISSUE TITLE
MissingSwitchDefault : False positive when `case null` in switch labeled statement group

### ISSUE THREAD

**Issue by mahfouz72** [checkstyle/checkstyle#15123]
Check documentation : https://checkstyle.org/checks/coding/missingswitchdefault.html#MissingSwitchDefault

---
```
PS D:\CS\test> javac src/Test.java                                               
PS D:\CS\test> cat src/Test.java                                                 
public class Test {
    void m(E i) {
        switch (i) {  // violation
            case FIRST :
            case SECOND :
            case null :
                System.out.println("NULL first and second");
        }
    }
    enum E {
        FIRST, SECOND
    }
}
PS D:\CS\test> cat config.xml                                                    
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
  <module name="TreeWalker">
            <module name="MissingSwitchDefault"/>
        </module>
</module>
PS D:\CS\test> java  -jar checkstyle-10.17.0-all.jar -c config.xml  src/Test.java
Starting audit...
[ERROR] D:\CS\test\src\Test.java:3:9: switch without "default" clause. [MissingSwitchDefault]
Audit done.
Checkstyle ends with 1 errors.
PS D:\CS\test> 

```

---
I expect no violation because there is a case null in the switch so exhaustiveness is being checked. the issue is that the check doesn't detect case null if it is in a labeled group.

**Comment by rnveach** [comment]
@mahfouz72 This isn't linked to a tracker issue. 

> I expect no violation because there is a case null in the switch so exhaustiveness is being checked.

Can you provide something that says this is the case, a switch must be "exhaustive" when it has this `null` case?

**Comment by rnveach** [comment]
I hid my above questions. This is supported, but part of support was missed. So this isn't anything new.
